ts/packages/ts/src/commands/EvalCommand.ts

```diff
@@ -36,7 +36,14 @@ export class EvalCommand extends Command<EvalCommandReturn> {
   }
 
   private evalFromConfigsOverride(result: EvalCommandReturn): EvalCommandReturn {
```

Comment:
I strongly feel like this method should remain untouched and `validateResult` should be responsible for throwing the `required` error which it already does. Is there any specific reason why we need to duplicate the validation here?

Reply:
done